client/hack/README.md

```diff
@@ -95,3 +99,68 @@ Update the restoreSize property to use type string only:
 
 ```
 
+* Because we need to serve both v1 and v1beta1 snapshot APIs, we need to make a copy of the generated v1 schema, paste it below and change the version to v1beta1 under `versions:` of each generated yaml file. See `snapshot.storage.k8s.io_volumesnapshots.yaml` as an example. `served` is true for both v1beta1 and v1. `storage` is true for v1beta and false for v1.
```

Comment:
I am not sure I fully understand this phrase:
"Because we need to serve both v1 and v1beta1 snapshot APIs, we need to make a copy of the generated v1 schema, paste it below and change the version to v1beta1 under `versions:` of each generated yaml file."

Do you mean after the v1 yaml files have been generated., the 'spec' piece needs to be copied to its corresponding yaml file? what does "change to v1beta1 under 'versions:'" mean? change v1. to v1beta1?

Reply:
I mean we need to copy the generated v1 section and pasted below for v1beta1.  Maybe we should just copy from the originally generated v1beta1 manifest?